apps/portal/js/wizard.js

```diff
@@ -146,7 +154,7 @@ $(".pager .finish").click(function() {
             type: $("#dsList option:selected").attr("data-type"),
             filter: $("#txtFilter").val(),
             columns: columns,
-            maxUpdateValue: 10,
+            maxUpdateValue: defaultMaxValue,
```

Comment:
@lasanthafdo 
wizard.js will generate a file called in gadgetconf inside store gadget directory upon gadget creation where as default value 10 is saved. There user can change specify desired values. 

when maxUpdateValue is 0, chart it will ignore max value and draw the whole data set